easybuild/easyconfigs/c/Coreutils/Coreutils-8.23-GCC-4.9.2.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = "Coreutils"
```

Comment:
It was not installed by me for 2014a, I could check with Martijn, but I'm pretty sure it was installed for the bioinformatics guys.  They tend to want to use the latest shell tools, and sometimes they even have a very valid reason to do so ;)